src/Common/Core/Test/Shell/TestCoreShell.cs

```diff
@@ -8,13 +8,14 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Common.Core;
+using Microsoft.Common.Core.Services;
+using Microsoft.Common.Core.Settings;
 using Microsoft.Common.Core.Shell;
-using Microsoft.Common.Core.Telemetry;
-using Microsoft.Common.Wpf.Threading;
+using Microsoft.Common.Core.Threading;
 using Microsoft.UnitTests.Core.Threading;
+using NSubstitute;
 
-namespace Microsoft.R.Components.Test.Fakes.Shell {
+namespace Microsoft.Common.Core.Test.Shell {
```

Comment:
I'd prefer to have all fakes in a separate folder, otherwise it may be hard to find them, or misuse instead of stubs and visa versa